interface-definitions/service_ntp.xml.in

```diff
@@ -86,6 +156,18 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <leafNode name="ptp-transport">
```

Comment:
Please only use `ptp` as node name with help string: `Enable Precision Time Protocol (PTP) transport for this server`